.buildkite/scripts/steps/esql_grammar_sync.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
```

Comment:
Thoughts on this instead of relying on the `|| exit` statements?  To help avoid any statements accidentally missing it.
```suggestion
#!/usr/bin/env bash

set -euo pipefail
```

We can wrap `git diff --exit-code` in a set +e, set -e block for checking.

Reply:
I like it.

Reply:
Any reason not to set `x` as well?

Reply:
It may be fine here.  I skip it generally, to avoid logging cases using credentials.  I'll leave it up to you.